Sources/Core/AWSClientRuntime/Middlewares/Sha256TreeHashMiddleware.swift

```diff
@@ -1,9 +1,12 @@
 // Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0.
 
+import class Smithy.Context
```

Comment:
`HttpContext` was moved into the new `Smithy` module and renamed to `Context` since it is not specific to HTTP.